signingscript/config_example.json

```diff
@@ -2,12 +2,11 @@
     "signing_server_config": "server_config.json",
     "work_dir": "/tmp/work_dir",
     "artifact_dir": "/tmp/artifact_dir",
-    "signtool": "signtool",
```

Comment:
\o/

Reply:
We may be able to delete `ssl_cert` as well; I think that's for the signing servers.

Reply:
Good catch! I've removed ssl_cert in a followup.